src/components/layouts/FilterBar.tsx

```diff
@@ -16,6 +16,7 @@ import { CreateViewSettingsDTO } from '@/types/dto/viewSettings.dto'
 import { FilterOptions, FilterOptionsKeywords, IAssigneeCombined, IFilterOptions } from '@/types/interfaces'
 import { filterOptionsToAssigneeMap, filterTypeToButtonIndexMap } from '@/types/objectMaps'
 import { checkAssignee } from '@/utils/assignee'
+import { stopPendingFetch } from '@/utils/fetchSideEffects'
```

Comment:
I think this is creating another issue. The previous issue was about search result persisting when dropdown is toggled again. 
Now the issue is - dropdown doesn't show any result at all when I open it from the second time onwards. Please kindly check the video below - 

https://www.loom.com/share/fe1c7b01502046ffb628068d63804537?sid=94b29072-cb37-4623-aa5e-16277edf3ec2